attest/ake/src/responder.rs

```diff
@@ -144,9 +150,10 @@ where
         let mut verifier = Verifier::default();
         verifier.identities(&identities).debug(DEBUG_ENCLAVE);
 
-        // Parse the received IAS report
+        // Parse the received attestation evidence
         let remote_report = VerificationReport::decode(payload.as_slice())
```

Comment:
This is another confusing spot where I would expect we can get either a `VerificationReport` or a `DcapEvidence`.

Reply:
The responders are the enclaves. Because of this they will only ever provide one or the other.
It's only the clients(Initiator.rs) that needs to handle the legacy EPID and the newer DCAP.